README.md

```diff
@@ -278,6 +281,7 @@ $ yarn login
 
 ### Prepare the release
 
+- Inform on Slack channel #kit-ui about the release with the related CHANGELOG.
```

Comment:
J'aurais dit `- Inform the #kit-ui Slack channel` parce que j'ai l'impression qu'il faut toujours "inform someone or something about", pas juste "inform about".